src/pybind/mgr/dashboard/controllers/nfs.py

```diff
@@ -145,11 +145,11 @@ def create(self, path, cluster_id, pseudo, access_type,
             'fsal': fsal,
             'clients': clients
         }
-        ret, _, err = export_mgr.apply_export(cluster_id, json.dumps(raw_ex))
-        if ret == 0:
+        applied_exports = export_mgr.apply_export(cluster_id, json.dumps(raw_ex))
+        if not applied_exports.has_error:
             return self._get_schema_export(
                 export_mgr.get_export_by_pseudo(cluster_id, pseudo))
-        raise NFSException(f"Export creation failed {err}")
+        raise NFSException(f"Export creation failed {applied_exports.msg}")
```

Comment:
I don't see a `.msg` field in the [`AppliedExportResults` class](https://github.com/ceph/ceph/pull/46209/files#diff-d00d146481b43490d2e6e79fcf061bf8fbd7364614e01613320d6e2234025c02R161).

Reply:
Fixed it.